modules/user-access/pages/access-control-model.adoc

```diff
@@ -2,7 +2,7 @@
 :description: A brief explanation of TigerGraph's access control model.
 :page-aliases: roles-and-privileges.adoc
 
-TigerGraph supports both Role-based access control(RBAC) and Access Control Lists (ACL).
+TigerGraph supports both Role-based Access Control (RBAC) and Access Control Lists (ACL).
```

Comment:
Since the acronym is RBAC, we should capitalized _Based_

Reply:
On second thought, since both "role-based access control" and "access control list" are common nouns, neither should be capitalized.